sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_server_events_v1/query.sql

```diff
@@ -0,0 +1,31 @@
+SELECT
+  -- example logger expected input: fxa-auth-server
+  SPLIT(jsonPayload.logger, "-")[OFFSET(1)] AS fxa_server,
+  * REPLACE (
+    (
+      SELECT AS STRUCT
+        jsonPayload.* REPLACE (
+          (
+            SELECT AS STRUCT
+              jsonPayload.fields.* REPLACE (
+                TO_HEX(SHA256(jsonPayload.fields.user_id)) AS user_id,
+                TO_HEX(SHA256(jsonPayload.fields.device_id)) AS device_id
+              )
+          ) AS fields
+        )
+    ) AS jsonPayload
+  ),
+FROM
+  `moz-fx-fxa-prod.gke_fxa_prod_log.stdout`
```

Comment:
Given the [revelation that auth logs are ending up in `stderr` instead of `stdout`](https://mozilla.slack.com/archives/C4D5ZA91B/p1694535065462929?thread_ts=1694446817.562329&cid=C4D5ZA91B) I think we'll need an additional ETL to archive the events from `stderr` (also in staging), and then name these ETLs differently since this won't be all server events.  Maybe something like `fxa_gcp_stdout_events_v1` and `fxa_gcp_stderr_events_v1`?

Reply:
@kik-kik we also need to filter out some high-volume events to keep this consistent with the old ETL: https://github.com/mozilla/bigquery-etl/blob/8a97f747949dc87d9b5425d82776b2c1626aca2e/sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_auth_events_v1/query.sql#L43-L62